proposal-selection.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: default
+---
+
+# Proposal selection process
+
+This process is used when selecting major initiatives to incorporate into the baseline of The Good Docs Project. In particular, it is used when deciding upon proposals to sponsor.
+
+**Version:** 1.0
+
+**Last updated:** August 2020
+
+## When do we use this process?
+
+This process is initiated periodically and typically depends upon:
+
+*   Timing of current build cycles.
+*   Sufficient bandwidth available within the community.
+*   Budget availability.
+
+Timing will typically align with the [Season of Docs](https://developers.google.com/season-of-docs) schedule.
+
+## Call for proposals
+
+On initiation of this process, a call for proposals will be sent to public forums, which will at a minimum include our [announce email list](https://thegooddocsproject.groups.io/g/announce). The announcement will include references to:
+
+*   Expectations for proposals.
+*   [Prioritization guidelines](prioritization.html).
+*   Eligibility.
+*   Associated timelines.
+*   Budget availability (if applicable).
+*   How to submit the proposal.
+
+## Eligibility
+
+*   You should be actively involved in The Good Docs Project or a related community. (Note, involvement in our community is open to all.)
+*   You will need to have won the trust of this community in order to have them back your proposal.
+*   You will need a [mentor](roles.html) before you get an application approved.
```

Comment:
This isn't technically true in GSoD. You get a mentor *after* you are accepted as a GSoD sponsored participant.